tests/test_answersfile_templating.py

```diff
@@ -0,0 +1,58 @@
+from pathlib import Path
+
+import pytest
+
+import copier
+
+from .helpers import build_file_tree
+
+
+@pytest.fixture(scope="module")
+def template_path(tmp_path_factory) -> str:
+    root = tmp_path_factory.mktemp("template")
+    build_file_tree(
+        {
+            root
+            / ".copier-answers-{{module_name}}.yml": """\
+                # Changes here will be overwritten by Copier
+                [[ _copier_answers|to_nice_yaml ]]
+                """,
+            root
+            / "copier.yml": """\
+                _answers_file: .answers-file-changed-in-template.yml
```

Comment:
The real place where you want to template the name is here:

```suggestion
                _answers_file: .copier-answers-{{module_name}}.yml
```